app/serializers/project_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class ProjectSerializer < ActiveModel::Serializer
```

Comment:
The primary reason for having the serializers scoped under `V1` is because serialization is part of an API's signature. If serializers change in a breaking way then we must do that in a new version but often will want the old version still there if the previous API version is still in use.